files/usr/share/jeos-firstboot/jeos-firstboot-dialogs

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+menulist()
+{
+	list=()
+	local line
+	while read line; do
+		list+=("$line" '')
+	done < <("$@"||true)
+	[ -n "$list" ]
+}
+
+dialog_locale()
+{
+        default="en_US"
```

Comment:
Once `get_current_locale` is fixed, this can just be `default="$(get_current_locale)"`